abaco.go

```diff
@@ -435,27 +610,49 @@ func NewAbacoSource() (*AbacoSource, error) {
 
 // Delete closes the ring buffers for all AbacoRings.
 func (as *AbacoSource) Delete() {
-	for _, dev := range as.arings {
-		dev.ring.Close()
-	}
+	as.closeDevices()
 }
 
 // AbacoSourceConfig holds the arguments needed to call AbacoSource.Configure by RPC.
 type AbacoSourceConfig struct {
 	ActiveCards     []int
 	AvailableCards  []int
-	Unwrapping      bool // whether to activate unwrapping
-	UnwrapResetSamp int  // unwrap after this many samples (or never if ≤0)
+	HostPortUDP     []string // host:port pairs to listen for UDP packets
+	Unwrapping      bool     // whether to activate unwrapping
+	UnwrapResetSamp int      // unwrap after this many samples (or never if ≤0)
```

Comment:
doesnt this "reset the unwrap offset" not "unwrap" after a certain number of samples?

Reply:
Good point. I'll change comment here and at line 576 to say "unwrap resets after this many...".